### PR TITLE
Always use new BPM connector - fix executor response.

### PR DESCRIPTION
**Pull request by matejonnet**
### Checklist:

* [ ] Have you added a note in the [CHANGELOG wiki](https://github.com/project-ncl/pnc/wiki/Changelog) for your change if user-facing?
* [ ] Have you added unit tests for your change?

### CODE REVIEW

File: bpm/src/main/java/org/jboss/pnc/bpm/ConnectorSelector.java [hunk 1]
```diff
@@ -35,9 +35,14 @@ public static boolean useNewProcess(BpmTask task) {
...
-                return true;
-            }
+            return useNewProcessForBuild(genericParameters);
+        }
+        return false;
+    }
+
+    public static boolean useNewProcessForBuild(Map<String, String> genericParameters) {
+        if (true || genericParameters.getOrDefault(GENERIC_PARAMETER_KEY, "").equals(RHPAM)) {
+            return true;
```
Comment (janinko) [inline bpm/src/main/java/org/jboss/pnc/bpm/ConnectorSelector.java:45]: you don't need if, you can just return the value of the condition;

Comment (janinko) [reply bpm/src/main/java/org/jboss/pnc/bpm/ConnectorSelector.java:45]: BTW there is plan to change it in the future to not always return true?

**Review by janinko — CHANGES_REQUESTED** [review]

File: bpm/src/main/java/org/jboss/pnc/bpm/ConnectorSelector.java (hunk 1 above)
Comment (matejonnet) [reply bpm/src/main/java/org/jboss/pnc/bpm/ConnectorSelector.java]: "true or" is there for easy roll-back if we need it, at some point we have to clean it up.

**Comment by matejonnet** [comment]
Re-test this please!

**Pull request merged**